CodeBeam.MudBlazor.Extensions/Components/SignaturePad/MudSignaturePad.razor

```diff
@@ -0,0 +1,56 @@
+﻿@using Microsoft.JSInterop
+@using MudBlazor.Charts
+@using MudBlazor.Utilities
+@using System.Text
+@namespace MudExtensions
+@inject IJSRuntime JsRuntime
+@inherits ComponentBase
+
+<MudPaper Elevation="4" Class="ma-4">
```

Comment:
We need a flexible option instead of hardcoded `ma-4`. You can prepare new parameter like `OuterClass` or make a classname like `mud-signature`

Reply:
Added an outer class and elevation parameter.